sjk-core/src/test/java/org/gridkit/jvmtool/cmd/StackSampleAnalyzerCmdTest.java

```diff
@@ -0,0 +1,49 @@
+package org.gridkit.jvmtool.cmd;
+
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DateFormater;
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DecimalFormater;
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.MemRateFormater;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.TimeZone;
+
+public class StackSampleAnalyzerCmdTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testToStringDateFormater() {
+    Assert.assertEquals("", new DateFormater(null).toString(null));
+    Assert.assertEquals("2019.02.18_17:17:14", new DateFormater(TimeZone.getDefault()).toString(1550510234000L));
+  }
+
+  @Test
+  public void testToStringDateFormaterException() {
+    thrown.expect(NullPointerException.class);
```

Comment:
If you have just one test that uses `thrown.expect`, you can have it as the last test in a method with other tests (i.e. these two lines could be appended to `testToStringDateFormater`). Again, maybe it's clearer as it is, I'm not sure.

Reply:
I've combined them and it makes the file neater.